ros/processor/inventory_events_consumer.py

```diff
@@ -131,6 +131,7 @@ def host_create_update_events(self, msg):
         if (
                 msg['platform_metadata'] is None
                 and msg['type'] == 'updated'
+                and msg['host']['system_profile']['cloud_provider']
         ) or 'is_ros' in msg['platform_metadata']:
```

Comment:
Due to this "or" condition there can be one edge case in which hosts(VM, baremetal) will get processed. 
If user had run ros ansible playbook on baremetal host than puptoo will set `is_ros=True` for that host's insights report and ros-backend will process it.
Might be I am overthinking but possible.

Reply:
@patilsuraj767 In that case wouldn't that be a putpoo thing to sort out?

Reply:
Yes, this can be fixed from puptoo side.
Approving the PR. Thanks @saltgen

Reply:
One more thing, I remembered in the past @kgaikwad mentioned for few **update** message like update of display_name full host object is not passed in kafka topic. In that case this `msg['host']['system_profile']['cloud_provider']` will return null.

Reply:
Checking, thanks @patilsuraj767

Reply:
Couldn't look into this issue yet, cause of an existng issue with host-inventory, would be great if someone can help me with the verification and changes if any, thanks!